smarthost/etc/e-smith/templates/etc/postfix/tls_policy/30smarthost-policy-sender

```diff
@@ -1,13 +1,34 @@
+
```

Comment:
is this new line really useful? :thinking:

Reply:
done